content/en/developers/ide_integrations/idea/_index.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Datadog Plugin For IntelliJ IDEA
+kind: documentation
+disable_toc: false
+is_beta: true
+further_reading:
+- link: "/account_management/api-app-keys/"
+  tag: "Documentation"
+  text: "Find out more about API and application keys."
+- link: "/getting_started/profiler/"
+  tag: "Documentation"
+  text: "Getting started with Continuous Profiler."
+---
+
+{{< beta-callout url="#" btn_hidden="true">}}
+  The Datadog plugin for IntelliJ IDEA is in Public Beta. It is intended for customers that use the <a href="https://docs.datadoghq.com/profiler/#pagetitle">Continuous Profiler</a> for their Java services. If the plugin stops working unexpectedly, check for plugin updates.
+{{< /beta-callout >}}
+
+## Overview
+
+The Datadog plugin for IntelliJ IDEA helps improve software performance by providing meaningful code-level insights in the IDE based on real-time observability data. The plugin helps reduce latency and lower cloud costs by highlighting code lines that:
+- consume the most CPU
+- allocate the most memory
+- spend the most time on locks, disk I/O, socket I/O, and so on
+
+{{< img src="/developers/ide_integrations/idea/overview.png" alt="The Datadog tool window open in IDEA" style="width:100%;" >}}
+
+The plugin features:
+- Top List: Identify methods that consume the most resources
+- Flame graph: visualize aggregated profile data
+- Find resource consumption broken down by method name and line number
+- Insights available from various staging and production environments
+
+## Requirements
+
+- **A Datadog account**: The plugin requires a Datadog account. If you're new to Datadog, go to the [Datadog website][4] to learn more about Datadog's observability tools and sign up for a free trial.
+- **Continuous Profiling**: To display code-level insights, the plugin requires Continuous Profiling instrumented on your Java Services. For more information, see [Getting Started with the Continuous Profiler][3].
+
+## Setup
+
+### Install the Datadog plugin
+
+1. Click **Plugins** and search for `Datadog`.
+1. Click **Install** to download and install the plugin in your IDE.
+1. If you receive a prompt notifying you that Datadog is a third-party plugin, click **Accept**.
+1. Click **Restart IDE** to restart IDEA.
+
+{{< img src="/developers/ide_integrations/idea/datadog-plugin1.png" alt="The Datadog plugin" style="width:100%;" >}}
+
+### Add Datadog credentials
+
```

Comment:
Do we want to add a small banner that this subject to change after the BETA?

Reply:
Could you clarify? What is subject to change?